.gitignore

```diff
@@ -173,3 +173,4 @@ test/data/*.json
 test/data/pipeline/*.json
 test/data/filters/*.json
 test/data/plang/*.json
+doc/workshop/exercises/workshop-updates.md
```

Comment:
Why is this file added to `.gitignore`  I would think this file we want to be in the repo?